### DISCUSSION TITLE
"Error" message on VGA screen at power on

### DISCUSSION THREAD

**Discussion by ElectricDream** [thehighnibble/thehighnibble.github.io#72] [S-132 add-on for IMSAI & Cromemco replicas]
Hi !

I just finished the assembly. All solders look clean, i added the "S132.ini=1" line in the boot.conf file.  I connected the add-on card to the IMSAI with VGA screen and USB keyboard. After power on, i can see on the screen the message "Host found", then "Wait", and ... "Error".
Now i have a blinking cursor on the screen, keyboard is initalized ( num lock works), but no keypress are registred (just a beep at each keystroke).

Before i'll recheck (again) all my solders with my multimeter, is there any way to access a log of some kind? Any advices?

Regards,

**Comment by GrantMeStrength** [comment]
The "Error" is normal because nothing has been saved to the internal memory yet.

Try the <ctrl><alt><F6> sequence to enter the set-up menu. <Alt><SysReq> or <Alt><PrintScreen> will also bring up settings for the Terminal.

USB keyboards are very picky. If your keyboard has a built-in hub, it'll probably not work. Try the simplest USB keyboard you have.
I actually get best results from a wireless keyboard (the USB wireless dongle plugged in) and a PS/2 keyboard connected via a PS/2 keyboard/mouse to USB adapter.

**Reply by ElectricDream** [reply]
Thanks for your reply. This may be a dumb question, but how do you enter the set-up menu? I can't find the key combination. Apart from the BOM and the build notes, is there any other documentation available regarding the S-132 add-on card? 

Thank you in advance for your help.

**Reply by GrantMeStrength** [reply]
The other 132 card shares a lot of firmware, so is applicable - https://thehighnibble.com/vt132/operation/vt100/operator/#background

**Reply by ElectricDream** [reply]
Thank you ! So it was all there! The build notes and the BOM for the S-132 contain limited information, and I was somewhat confused . There is even a dedicated video about how to properly solder the USB interface for the VT-132 ! How could I miss this? I wasn't aware of the VT-132.

@dmcnaugh if you read this, maybe adding a little link to the VT-132 highly detailed information pages at the end of the Build notes/BOM could help lost people like me.

Again, thank you

**Reply by ElectricDream** [reply]
It's me again. 

So i managed to get to the setup screen with ALT+PRTSCR, i set everything to my taste (keyboard layout, color, etc..), saved the setup to the NVR. I can access the system menu, mount and eject Disks, view the Library and System infos. I can switch to the VT100 TTY (CTRL+ALT+F1), but when i boot CPM pushing RUN switch,  nothing appends : no A: prompt, no echo when i'm typing. I checked, i'm in ONLINE mode. When i switch to LOCAL mode, I have keypresses are displayed on the VT100 screen. But when i switch back to ONLINE, nothing happens. 

Any ideas?

**Reply by GrantMeStrength** [reply]
Did you set up everything as detailed here? https://thehighnibble.com/s132/configure/#configure-the-imsai8080

**Reply by ElectricDream** [reply]
I completely missed that page too, I must be really tired ... Everything is working now. Thank you again !
